src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  background-color: var(--iui-color-background-3);
+  border-top: 1px solid var(--iui-color-background-border);
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+@mixin iui-gdpr-banner-title {
+  @include iui-dialog-title;
+}
+
+@mixin iui-gdpr-banner-title-bar {
+  @include iui-dialog-title-bar;
+}
+
+@mixin iui-gdpr-banner-content {
+  @include iui-dialog-content;
+}
+
+@mixin iui-gdpr-banner-button-bar {
+  @include iui-dialog-button-bar;
+}
+
+//@mixin iui-anchor {
+//  @include iui-anchor;
+//}
+
+//.iui-gdpr-banner-cookie-policy {
+// color: var(--iui-color-foreground-primary);
+//}
```

Comment:
Is this leftover? unfinished?

Reply:
Oh yeah I forgot to take out those comments haha